src/types/redis_json.cc

```diff
@@ -117,4 +117,39 @@ rocksdb::Status Json::Get(const std::string &user_key, const std::vector<std::st
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status Json::StrAppend(const std::string &user_key, const std::vector<std::string> &paths, const std::string &value, std::vector<uint64_t> &append_cnt) {
```

Comment:
Path should be std::string?

Reply:
Get && Set function`s path is std::string，Can you explain your doubt?

Reply:
Yes, my point is should use std::string instead of const std::vector<std::string>.

Reply:
but, strAppend is support multipath

Reply:
It should be only one or none: https://redis.io/commands/json.strappend/

Reply:
`[path]` means option, I misunderstood... ,I thought means a array